catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/AccessControlAccessPlugin.java

```diff
@@ -52,19 +52,28 @@ public AccessControlAccessPlugin(SubjectIdentity subjectIdentity) {
   // Equivalent to doing a set intersection of the subject with the access-admin list
   private final Predicate<Metacard> subjectIsAccessAdmin =
       (newMetacard) ->
-          attributeToSet
+          ATTRIBUTE_TO_SET
               .apply(newMetacard, Security.ACCESS_ADMINISTRATORS)
               .contains(subjectSupplier.get());
 
+  private final Predicate<Metacard> subjectHasWritePerms =
+      (newMetacard) ->
+          ATTRIBUTE_TO_SET
+                  .apply(newMetacard, Security.ACCESS_GROUPS)
```

Comment:
Need to make sure we check if the users associated _groups_ are in the access_groups list